Quadriga/src/main/resources/locale/errormessages.properties

```diff
@@ -1,6 +1,6 @@
+error_message_400 = The server cannot or will not process the request due to a client error.
 error_message_401 = Unauthorized Access. Please check login credentials.
 error_message_403 = Not authorized to access this page.
-error_message_404 = Resource not found
-error_message_405 = Database is down!! People are working on it
-error_message_406 = Unknown error occurred please contact the administrator
-
+error_message_404 = Resource not found.
+error_message_406 = Unknown error occurred please contact the administrator.

```

Comment:
This was written in QuadrigaRestExceptionHandler
if(errorcode==0)
errorcode=406;
To may be say that if there was no error code set, give this value.
This is not thrown from anywhere though.